smithy-lang/smithy-examples
# Add a basic web application tutorial using smithy

#### Background
- This adds `smithy-tutorial`, a multi-component project that uses smithy for client and server generation
- The web application uses the generated client to make calls to a server implementation using the server-sdk
- This example uses typescript, both for client and server

#### Testing
- Pull locally
- Start the server: `make dev-server`
- Start the app: `make dev-app`

---
By submitting this pull request, I confirm that my contribution is made under the terms of the MIT-0 license.
